docs/UserGuide.md

```diff
@@ -237,11 +242,18 @@ Examples:
 * `deleteexam -id 888888 -exam Computing -date 2024-04-27 -time 09:00` would delete an exam of Computing with date of 2024-04-27 and time of 09:00 from a student whose id is 888888 if the student exists.
 
 ### Logging the Lessons of a student: `log`
-This will log the lessons of a student. The time field of the log entry will be the system time when the log was added.
-Format `log -id {ID} -hours {HOURS} -content {CONTENT} -style {LEARNING STYLE} -notes {NOTES}
+This will add a log to the lessons of a student. The time field of the log entry will be the *system time* when the log was added.
+Format: `log -id {ID} -hours {HOURS} -content {CONTENT} -style {LEARNING STYLE} -notes {NOTES}`
+<box type="info" seamless>
+Note: Hours need not be an integer input. Values like `2 hours 45 mins`, `2+ hours` are valid as well.
+</box>
 Examples:
-* `log -id 1 -hours 2 -content English Comprehension -style Visual -notes Great improvement!`. would log a lesson for ID 1 noting that it had great improvement in English Comprehension, it also logs the learning style of the student
+* `log -id 000001 -hours 2 -content English Comprehension -style Visual -notes Great improvement!`. would log a lesson for ID 1 noting that it had great improvement in English Comprehension, it also logs the learning style of the student
```

Comment:
Just a nit, should we put 000001 instead of 1 to match the command ?

Reply:
Yup! Nice catch